indico/modules/events/persons/controllers.py

```diff
@@ -151,17 +148,38 @@ def _checkParams(self, params):
 
     def _process(self):
         person_ids = request.form.getlist('person_id')
-        recipients = {p.email for p in self._find_event_persons(person_ids) if p.email}
-        form = EmailEventPersonsForm(person_id=person_ids, recipients=', '.join(recipients))
+        recipients = self._find_event_persons(person_ids)
+        if request.args.get('not_invited_only') == '1':
+            recipients = {p for p in recipients if p.invited_dt is None}
+        default_body = ''
+        default_subject = ''
+        disable_until_change = True
+        if request.args.get('no_account') == '1':
+            tpl = get_template_module('events/registration/emails/pending_submitter.html', event=self.event_new)
```

Comment:
This file doesn't belong in the registration module - "registration" has nothing to do with registering indico accounts but with registering for an event. While user registration is handled by the "auth" module, this email is part of "persons" so it should be in there.

Not sure about the naming of the file either. What about `events/persons/emails/invitation.html`?